src/OrchardCore.Modules/OrchardCore.Email/Extensions/OrchardCoreBuilderExtensions.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.Extensions.Configuration;
+using OrchardCore.Environment.Shell.Configuration;
+using OrchardCore.Email;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class OrchardCoreBuilderExtensions
+    {
+        public static OrchardCoreBuilder ConfigureEmailSettings(this OrchardCoreBuilder builder)
```

Comment:
I know some may prefer `ConfigureSmtpSettings` I just use the current name for consistency, also the email module might have POP, IMAP in the upcoming future